java/code_to_gast/java_router.py

```diff
@@ -19,7 +19,8 @@ def node_to_gast(node):
             return {"type": "bool", "value": 0}
         else:
             return "Unsupported prim"
-    
+    elif type(node) == str:
```

Comment:
What is this doing?

Reply:
Java doesn't have it's own type for variable names, so if we pass in just a string then we are going to assume its a variable name

Reply:
Interesting, so what was happening before this change?

Reply:
In the variable assignment section it is doing the same thing just not going through the router